src/main/java/seedu/address/logic/parser/DeleteTagCommandParser.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.DeleteTagCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new DeleteCommand object
+ */
+public class DeleteTagCommandParser {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteCommand
+     * and returns an DeleteCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public DeleteTagCommand parse(String args) throws ParseException {
+        try {
+            return new DeleteTagCommand(args);
+        } catch (IllegalValueException ive) {
```

Comment:
I think it would be better if we try to locate the parsing error within the parser.
How about importing isValidTagName from the Tag class and test for the validity of the argument right in the parser?